### DISCUSSION TITLE
j avais un prbl au niveau fast api n 'est pas marché

### DISCUSSION THREAD

**Discussion by tbougares** [fastapi/fastapi#4872] [Questions] [answered]
j avais un problème au niveau fast api n 'est pas marché. any solution

**Comment by tiangolo** [comment, marked as answer]
Sorry, I don't speak French. Please create a new question following the template, creating a self-contained example, etc, and writing in English, so that we all can understand.

> Sorry for the long delay! 🙈  I wanted to personally address each issue/PR and they piled up through time, but now I'm checking each one in order.
